flink-autoscaler/src/test/java/org/apache/flink/autoscaler/JobVertexScalerTest.java

```diff
@@ -1004,6 +1036,38 @@ public void testNumPartitionsAdjustment() {
                         parallelismUpperLimit,
                         eventCollector,
                         context));
+
+        assertEquals(
+                199,
+                JobVertexScaler.scale(
+                        vertex,
+                        24,
+                        List.of(),
+                        199,
+                        256,
+                        4,
+                        parallelismLowerLimit,
+                        parallelismUpperLimit,
+                        eventCollector,
+                        context));
+
+        Configuration conf = context.getConfiguration();
+        conf.set(
+                AutoScalerOptions.SCALING_KEY_GROUP_PARTITIONS_ADJUST_MODE,
+                NumKeyGroupsOrPartitionsParallelismAdjuster.Mode.MAXIMIZE_UTILISATION);
+        assertEquals(
+                100,
```

Comment:
```suggestion
                MAXIMUM_UTILISATION_PARALLELISM,
```

Its not the number that matters but what it means.